lib/dependencies/LocalModulesHelpers.js

```diff
@@ -40,4 +20,29 @@ function lookup(parent, mod) {
 	}
 
 	return path.join("/");
+};
+
+class LocalModulesHelpers {
+
+	static addLocalModule(state, name) {
+		if(!state.localModules) state.localModules = [];
+		var m = new LocalModule(state.module, name, state.localModules.length);
+		state.localModules.push(m);
+		return m;
+	}
+
+	static getLocalModule(state, name, namedModule) {
+		if(!state.localModules) return null;
+		if(namedModule) {
+			// resolve dependency name relative to the defining named module
+			name = lookup(namedModule, name);
+		}
+		for(var i = 0; i < state.localModules.length; i++) {
+			if(state.localModules[i].name === name)
+				return state.localModules[i];
+		}
+		return null;
+	}
 }
+
+module.exports = LocalModulesHelpers;
```

Comment:
This change makes `module.exports` constructible.

Instead of using a class, this should be written as plain functions. Something like:

```js
exports.addLocalModule = (state, name) => {
  /* ... */
}
exports.getLocalModule = (state, name, namedModule) => {
  /* ... */
}